src/collections/news/2023-04-15-Baby-Bliss-Bot-AI-Techniques.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Baby Bliss Bot - Experiments with AI
+slug: clusive-latest-features
+date: "2023-04-15"
+excerpt: Exploring AI Machine Learning models with Bliss symbols.
+---
+A number of different artificial intelligence machine learning systems were
+explored in order to acquire a basic knowledge of the systems, how to train
+them, and what they produce, as well as an idea of their value to the
+[Baby Bliss Bot Project](https://github.com/fluid-project/floeproject.org/blob/eb04bf9b0774f452145376f3495933eb3ce81c10/src/collections/news/2023-04-12-BBB-Intro.md).
+ __TODO: Fix link after referenced blog post is merged.__
+
+## StyleGAN: StyleGAN2-ADA and StyleGAN3
+
+A GAN (Generative Adversarial Network) is an AI that generates fake content
```

Comment:
AI -> a machine learning framework for generative AI

Reply:
Agreed, but "generate/generative/generates" is a bit repetitive:
 "A GAN (Generative Adversarial Network) is a machine learning framework for generative AI that generates fake content ...".

I went with:
 "A GAN (Generative Adversarial Network) is a machine learning framework for a type of AI that generates fake content ...".